server/webhook.go

```diff
@@ -22,6 +23,11 @@ const (
 	commentCreated = "comment_created"
 )
 
+const (
+	recipientTypeAssignee = "assignee"
+	recipientTypeReporter = "reporter"
```

Comment:
`recipientTypeAssignee` isn't used anywhere. What is the use for introducing the `recipientType` property?

Reply:
can refer to this comment https://github.com/mattermost/mattermost-plugin-jira/pull/810#discussion_r760757276